app/components/ChartAnnotations/index.js

```diff
@@ -190,6 +209,9 @@ class ChartAnnotations extends Component {
     this.setState({
       showAnnotations: !this.state.showAnnotations,
     });
+    if (this.state.showAnnotations) {
+      setTimeout((() => this.beginRenderingAnnotations()));
```

Comment:
Why do you need set timeout?

Reply:
..thought i did.  guess not.  removed :)